ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+**Kind of issue**
+- [ ] Site request (add support for a new website)
+- [ ] Feature request (request new functionality)
+- [ ] Bug report (encountered problems with youtube-dl)
+- [ ] Other, namely ...
+
+**Run `youtube-dl --version` to check whether you have the latest version**
+- [ ] I've verified that I'm running the latest version available (2016.03.06)
```

Comment:
This should be on the very top. I'd word it differently:

Make sure you are using the **latest** version of youtube-dl. Latest youtube-dl vesion at the moment is **2016.03.06**. Read [our FAQ](https://yt-dl.org/update) if you have troubles updating.
[ ] I've verified that I'm running youtube-dl version 2016.03.06